draft-ietf-quic-tls.md

```diff
@@ -1426,8 +1426,8 @@ packet that contains the changed value.
 
 This mechanism replaces the TLS KeyUpdate message.  Endpoints MUST NOT send a
 TLS KeyUpdate message.  Endpoints MUST treat the receipt of a TLS KeyUpdate
-message as a connection error of type 0x10a, equivalent to a fatal TLS alert of
-unexpected_message (see {{tls-errors}}).
+message in a 1-RTT packet as a connection error of type 0x10a, equivalent to a
+fatal TLS alert of unexpected_message (see {{tls-errors}}).
```

Comment:
IIUC, KeyUpdate is forbidden by the the TLS spec, right? Can we add a sentence here that states taht?

Reply:
PTAL at b33c487. With the change, I think the text now aligns better with what we state for EOED ([section 8.3](https://quicwg.org/base-drafts/draft-ietf-quic-tls.html#name-removing-the-endofearlydata)).